packages/react-dom/src/__tests__/ReactDOMSelect-test.js

```diff
@@ -911,9 +911,7 @@ describe('ReactDOMSelect', () => {
     const container = document.createElement('div');
     const root = ReactDOMClient.createRoot(container);
     async function changeView() {
-      await act(() => {
```

Comment:
This triggers the interleave act warning but that should've already happened before since the dispatch is wrapped by act. So this is fixing something but not sure why this wasn't already erroring before.